### PR TITLE
feat(frontend): add confirm dialogs to actions

### PR DESCRIPTION
**Pull request by fkm-adfinis**
Some actions should avoid being executed by accident.

### CODE REVIEW

File: frontend/app/ui/subscriptions/confirm/controller.js [hunk 1]
```diff
@@ -17,12 +18,28 @@ export default class SubscriptionsConfirmController extends Controller {
...
+    try {
+      await UIkit.modal.confirm(
+        this.intl.t("page.subscriptions.confirm.prompt.accept", {
+          hours: order.duration.asHours(),
+          customer,
+          project,
+        })
+      );
+    } catch (error) {
+      return;
```
Comment (velrest) [inline frontend/app/ui/subscriptions/confirm/controller.js:32]: why is this in a try catch if you then dont handle the error / give an info to the user?

Comment (fkm-adfinis) [reply frontend/app/ui/subscriptions/confirm/controller.js:32]: The "error" is when the user cancels the confirmation. I agree that this is rather weird. I would have expected a boolean as with [window.confirm](https://developer.mozilla.org/en-US/docs/Web/API/Window/confirm). But this is [how UIkit implemented](https://getuikit.com/docs/modal#modal-dialogs) this.

So. There is no real error to handle.

Comment (velrest) [reply frontend/app/ui/subscriptions/confirm/controller.js:32]: Ah okay :ok_hand:

**Review by velrest — APPROVED** [review]

File: frontend/app/ui/subscriptions/confirm/controller.js (hunk 1 above)
Comment (fkm-adfinis) [reply frontend/app/ui/subscriptions/confirm/controller.js:32]: I guess the reasoning is that you have to have a catch anyway.

**Pull request merged**